modules/oe_translation_poetry/src/Form/PoetryCheckoutFormBase.php

```diff
@@ -387,10 +397,11 @@ protected function createRequestTitle(): string {
    */
   protected function handlePoetryResponse(ResponseInterface $response, FormStateInterface $form_state): void {
     if (!$response->isSuccessful()) {
-      $this->rejectJobs($response);
+      $this->rejectJobs($response, $form_state);
```

Comment:
New `rejectJobs` argument `$form_state` is used there only to get the jobs.
Getting the jobs from the queue is also being done in the method just after.

Then it's better to just get first the jobs here and pass them as a `rejectJobs` argument.

Reply:
yes, that's fine